### DISCUSSION TITLE
Using Reagraph with Melange

### DISCUSSION THREAD

**Discussion by faseidl** [reaviz/reagraph#276] [Q&A]
Has anyone used Reagraph from a Melange application?

I tried installing Reagraph via npm, but have been unable to get Melange to recognize it. I see that the `reagraph` package does exist in the node_modules directory, but an `open Reagraph` in Melange results in a "module not found" error.

Thank you for any insights.

**Comment by amcdnl** [comment]
Can you provide a example repo or something for us to start with? I'm not familiar with Melange.

**Reply by faseidl** [reply]
Thanks for the quick reply. 
Melange is a ReasonML-based framework for building React apps. In theory, this should be compatible with Reagraph, but there may be some binding magic that I'm missing or doing wrong.

https://github.com/melange-re/melange-re.github.io

**Comment by faseidl** [comment]
I found a solution:
![image](https://github.com/user-attachments/assets/1fde962d-e9e5-468c-a7dd-eb5de5124502)
